discord-roles.md

```diff
@@ -36,6 +36,7 @@ The ownership of the Discord is held by a special account: `@neoforged.admin`. I
 | All Vanity Roles       | All team members (optionally)     | The only roles with colors.                                       |        :x:         | 1 for each MC Color | 1 for each hoisted role + combined |
 | Nitro Booster          | Nitro boosters                    | Handled by the Discord integration.                               |        :x:         |        None         |         Nitro Booster Icon         |
 | All Bot Roles          | All bots                          | Discord-demanded integrations.                                    |        :x:         |        None         |                Gear                |
+| Collaborator           | Special non-team members          | High-impact people. Grants access to #project-talk channel        |        :x:         |        None         |                None                |
```

Comment:
We have the roles list in order here, so I think the Collaborator role should be moved upwards, to be somewhere above Former Member and below Probation (I'm unsure whether it should be above or below Community Volunteers).